libethcore/BlockInfo.h

```diff
@@ -149,11 +149,10 @@ class BlockInfo
 
 	void clear();
 	void noteDirty() const { m_hashWithout = m_boundary = m_hash = h256(); }
+	void streamRLPFields(RLPStream& _s) const;
```

Comment:
`streamRLPFields` shouldn't be public API here. maybe add a friend to BlockInfo for the test class?